packages/replay/test/integration/flush.test.ts

```diff
@@ -472,10 +472,9 @@ describe('Integration | flush', () => {
     const TEST_EVENT = { data: {}, timestamp: BASE_TIMESTAMP + 100, type: 2 };
     mockRecord._emitter(TEST_EVENT);
 
-    await advanceTimers(DEFAULT_FLUSH_MIN_DELAY);
     await advanceTimers(160_000);
 
-    expect(mockFlush).toHaveBeenCalledTimes(2);
+    expect(mockFlush).toHaveBeenCalledTimes(1);
```

Comment:
This also shows the incorrect behavior, it was flushing twice before even though it shouldn't have.